components/board/pinwrappers/analogs.go

```diff
@@ -57,21 +58,22 @@ func (as *AnalogSmoother) Close(ctx context.Context) error {
 }
 
 // Read returns the smoothed out reading.
-func (as *AnalogSmoother) Read(ctx context.Context, extra map[string]interface{}) (int, error) {
+func (as *AnalogSmoother) Read(ctx context.Context, extra map[string]interface{}) (int, board.AnalogRange, error) {
 	if as.data == nil { // We're using raw data, and not averaging
-		return as.lastData, nil
+		return as.lastData, board.AnalogRange{}, nil
```

Comment:
This feels wrong: if we're using raw data and not averaging, that has nothing to do with the range, and we should still return the correct range, right? Why shouldn't we return `as.analogRange` here?

Reply:
my bad forgot to change this one